recipes/libsecret/all/conanfile.py

```diff
@@ -53,23 +55,25 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("glib/2.78.1", transitive_headers=True, transitive_libs=True)
-        if self._use_gcrypt:
+        self.requires("glib/2.78.3", transitive_headers=True, transitive_libs=True)
+        if self.options.crypto == "libgcrypt" or self.options.get_safe("with_libgcrypt") == True:
```

Comment:
```suggestion
        if self.options.crypto == "libgcrypt":
```

A deprecated option should not be part of the recipe flow.